lib/public/Calendar/CalendarExportOptions.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+namespace OCP\Calendar;
+
+/**
+ * Calendar Export Options
+ *
+ * @since 32.0.0
+ */
+class CalendarExportOptions {
```

Comment:
```suggestion
final class CalendarExportOptions {
```

Something to consider. It prevents apps from extending the class.

Reply:
Done